code/game/objects/items/devices/lighting/marshalling wand.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/device/flashlight/marshallingwand
+	name = "marshalling wand"
+	desc = "A marshalling wand, used to direct air or space faring vessels during takeoff and landing."
+	icon_state = "marshallingwand"
+	item_state = "marshallingwand"
+	w_class = ITEMSIZE_SMALL
+	light_color = LIGHT_COLOR_RED
+	light_wedge = LIGHT_OMNI
+	brightness_on = 2
+	action_button_name = "Toggle Marshalling Wands"
+
+/obj/item/device/flashlight/marshallingwand/AltClick(mob/user)
+	if(!isturf(user.loc))
+		to_chat(user, "You cannot turn the light on while in this [user.loc].") //To prevent some lighting anomalities.)
+		return FALSE
```

Comment:
Add an use_check_and_message() at the beginning: the way this code is set up right now you can alt click it while unconscious from the other side of the room and it'd still change.

Also, add a span class to the to_chat here.

Reply:
Fixed!